TombEngine/Game/collision/collide_item.cpp

```diff
@@ -662,19 +662,24 @@ bool ItemPushItem(ItemInfo* item0, ItemInfo* item1, CollisionInfo* coll, bool en
 	int maxX = bounds.X2;
 	int minZ = bounds.Z1;
 	int maxZ = bounds.Z2;
+	int minY = bounds.Y1;
+	int maxY = bounds.Y2;
 
 	if (bigPushFlags & 1)
 	{
 		minX -= coll->Setup.Radius;
 		maxX += coll->Setup.Radius;
 		minZ -= coll->Setup.Radius;
 		maxZ += coll->Setup.Radius;
+		minY -= coll->Setup.Radius;
+		maxY += coll->Setup.Radius;
 	}
 
 	// Big enemies.
 	if (abs(deltaPos.x) > BLOCK(4.5f) || abs(deltaPos.z) > BLOCK(4.5f) ||
 		relDeltaPos.x <= minX || relDeltaPos.x >= maxX ||
-		relDeltaPos.z <= minZ || relDeltaPos.z >= maxZ)
+		relDeltaPos.z <= minZ || relDeltaPos.z >= maxZ ||
```

Comment:
Can you give an overview of this change? I suspect this has something to do with pushing item on vertical (Y) axis?

Reply:
This prevents an object's bounding box from essentially having infinite height. The player was getting pushed by blocks even if they were passing overhead.